multisig/proposals/vip-002/vip-002-ethereum.ts

```diff
@@ -0,0 +1,503 @@
+const IR_BASE200_SLOPE2000_JUMP30000_KINK5000 = "0x87C427b00C89E82064B32Ca63c9E983fedD3e53e";
+
+// IL configuration
+export const vip002 = () => {
+  return makeProposal([
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setCollateralFactor(address,uint256,uint256)", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMarketSupplyCaps(address[],uint256[])", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMarketBorrowCaps(address[],uint256[])", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setLiquidationIncentive(uint256)", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setCloseFactor(uint256)", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMinLiquidatableCollateral(uint256)", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "supportMarket(address)", POOL_REGISTRY],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setCloseFactor(uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setReduceReservesBlockDelta(uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setCollateralFactor(address,uint256,uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setLiquidationIncentive(uint256)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMarketBorrowCaps(address[],uint256[])", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMarketSupplyCaps(address[],uint256[])", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setActionsPaused(address[],uint256[],bool)", ETHEREUM_MULTISIG],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMinLiquidatableCollateral(uint256)", ETHEREUM_MULTISIG],
+    },
```

Comment:
Missing function in the Comptrollers: `setForcedLiquidation(address,bool)`

We should authorize the ETHEREUM_MULTISG to execute it:
```
    {
      target: ACM,
      signature: "giveCallPermission(address,string,address)",
      params: [ZERO_ADDRESS, "setForcedLiquidation(address,bool)", ETHEREUM_MULTISIG],
    },
```